### PR TITLE
Update verisure.markdown

### PR DESCRIPTION
**Pull request by prokon**
Updated based on PR: https://github.com/home-assistant/core/pull/70375

## Proposed change
Added new State attribute in PR above, include in documentation

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/70375
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/verisure.markdown [hunk 1]
```diff
@@ -87,3 +87,19 @@ automation:
 | disable_autolock | Disables autolock function for a specific lock. |
 | enable_autolock | Enables autolock function for a specific lock. |
 | smartcam_capture | Capture a new image from a specific smartcam. |
+
+## Lock
+| State | Description |
+| ------- | ----------- |
+| locked | Lock is locked |
+| unlocked | Lock is unlocked |
```
Comment (frenck) [inline source/_integrations/verisure.markdown]: These describe states of the Lock entity, which are be handled by the Lock documentation, and should not be documented the integration-specific documentation.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — CHANGES_REQUESTED** [review]
This PR targets the `current` branch, but its feature added depends on a next Home Assistant release. Please change the target branch for this PR to `next` and git rebase the PR.

File: source/_integrations/verisure.markdown (hunk 1 above)
Comment (prokon) [reply source/_integrations/verisure.markdown]: I see, good point. Ok now?

**Review by frenck — APPROVED** [review]
Fix the base.

Thanks for the PR @prokon 👍

**Pull request merged**